lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -224,11 +224,16 @@ class CategoryRenderer {
 
     // Add audits without a group first so they will appear first.
     const notAGroup = 'NotAGroup';
+    const hiddenGroup = 'hidden';
     grouped.set(notAGroup, []);
 
     for (const auditRef of auditRefs) {
       const groupId = auditRef.group || notAGroup;
       const groupAuditRefs = grouped.get(groupId) || [];
+
+      // Do not render audit if it is in the 'hidden' group.
+      if (groupId === hiddenGroup) continue;
```

Comment:
Just an option, but you could also do this similarly to what you did in `pwa-category-renderer` and move this check up the stack to `render()` (I realize that I was the person who told you this was a good spot for the check, sorry :).

Should just be the same-ish thing:

```js
// Do not render audit if it is in the 'hidden' group.
if (auditRef.group === 'hidden') continue; 
```
at the top of the loop https://github.com/GoogleChrome/lighthouse/blob/25de337597ed0a299de5bffd1597aec30d35645f/lighthouse-core/report/html/renderer/category-renderer.js#L464-L465